modules/core/src/test/resources/user.properties

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
I suggest to move this file to a package with a test using it. `user.properties` file in a resources root directory can look confusing.

Reply:
Yep, you are absolutely right. Fixed.